pkg/generate/git/git_test.go

```diff
@@ -112,7 +112,7 @@ func TestParseRepository(t *testing.T) {
 		}
 
 		if !reflect.DeepEqual(*got, want) {
-			t.Errorf("%s: got %v, want %v", scenario, got, want)
+			t.Errorf("%s: got %#v, want %#v", scenario, got, want)
```

Comment:
:+1: agreed and updated.
we're comparing *got to want, not the usual got and want.